### ISSUE TITLE
About loss value after 12000 steps.

### ISSUE THREAD

**Issue by YinRui1991** [rohitgirdhar/AttentionalPoolingAction#4]
@rohitgirdhar Thanks for your excellent code! I tested the trained models on the validation set, but the result of attentional model is:  Mean AP: 0.0408322168145 Accuracy: 0.0651116199199; result of the pose attentional model is: Mean AP: 0.0203709144522 Accuracy: 0.0435031482541. These two models are got after 12000 training steps without pretrained models. For attentional model, the loss decreased from 32.7 to 27.2, and for pose attentional model, the loss decreased from 32.7 to 30.2. Can you share the final loss of these two models? Maybe I need train more steps to decrease the loss.

**Comment by rohitgirdhar** [comment]
Thanks for the feedback!
These loss/mAP values indicate something is going wrong. The loss should start around 8~10 I think. Can you try running the pre-trained models I provide to see what performance you get with those?

**Comment by YinRui1991** [comment]
@rohitgirdhar Thanks for your reply. I will download the pre-trained models and start training with them.

**Comment by YinRui1991** [comment]
@rohitgirdhar Thanks for your advice, I run the pre-trained models and get the same performance showed in readme.  Then I start training with pre-trained models and the loss value is about 2. I think my problem has been solved. Thanks again!

**Comment by rohitgirdhar** [comment]
Great, thanks for checking! However, you should be able to train directly from ImageNet models as well (as shown in [training](https://github.com/rohitgirdhar/AttentionalPoolingAction#training) instructions). Make sure you are using the right initialization models.

**Comment by ghost** [comment]
Hi @rohitgirdhar . Thanks for the code and . Do you have some comparisons of the accuracies with other implementations ? In the paper, I could find mAP values compared. Do you think if accuracies can be improved further as I am focusing mainly on classification accuracy.

Thanks.
